examples/metrics.json

```diff
@@ -29,7 +29,7 @@
             {
               "name": "my.counter",
               "unit": "1",
-              "description": "I'm a Counter",
```

Comment:
Can we change simply to "I am a Counter"?

Reply:
Done :)